.github/workflows/jekyll.yml

```diff
@@ -40,6 +40,12 @@ jobs:
         run: bundle exec jekyll build --baseurl "${{ steps.pages.outputs.base_path }}"
         env:
           JEKYLL_ENV: production
+      - name: Make zipped archive of the site available
+        run: |
+          cd _site/
+          zip -r duckdb-docs.zip * -x "docs/archive/*"
```

Comment:
You're tagging this file using the currentduckdbversion variable, but then excluding the files for that version as they're in the archive folder?

Reply:
I am excluding the archives but then I'm adding the latest one of them back:
https://github.com/duckdb/duckdb-web/pull/1112/files#diff-60ac246cc02819dea1007076ba57241caa7c7d1e8232a5179124d4e63bb366f8R47-R48

So the zip has latest and stable, which is IMHO a good combination to have.